src/main/java/thePackmaster/powers/weaponspack/WeaponPower.java

```diff
@@ -0,0 +1,150 @@
+package thePackmaster.powers.weaponspack;
+
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.AbstractCard.CardType;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.cards.DamageInfo.DamageType;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import com.megacrit.cardcrawl.orbs.AbstractOrb;
+import thePackmaster.orbs.weaponspack.AbstractWeaponOrb;
+import thePackmaster.powers.AbstractPackmasterPower;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class WeaponPower extends AbstractPackmasterPower {
+
+    public static final String POWER_ID = makeID("WeaponPower");
+    private static final PowerStrings powerStrings = CardCrawlGame.languagePack.getPowerStrings(POWER_ID);
+    public static final String NAME = powerStrings.NAME;
+    public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
+
+    public WeaponPower(AbstractCreature owner) {
+        super(POWER_ID, NAME, PowerType.BUFF, false, owner, -1);
```

Comment:
This is just a system power, it'd be nice if you used the neutral type so it doesn't interact with other powers. Could also make this invisible by implementing the InvisiblePower interface from StSLib
```suggestion
        super(POWER_ID, NAME, NeutralPowertypePatch.NEUTRAL, false, owner, -1);
```

Reply:
Changed to Neutral and made it invisible.